web/package.json

```diff
@@ -24,6 +24,7 @@
     "open-graph": "^0.2.6",
     "react": "17.0.2",
     "react-dom": "^17.0.2",
+    "react-dropdown": "^1.11.0",
```

Comment:
"react-datepicker" wasn't added 😅 If you run `npm i react-datepicker` again it'll add it to the package though